app/bundles/LeadBundle/Entity/LeadDeviceRepository.php

```diff
@@ -156,15 +156,14 @@ public function isAnyLeadDeviceTracked(Lead $lead)
      */
     public function getLeadDevices(Lead $lead)
     {
-        $qb = $this->getEntityManager()->getConnection()->createQueryBuilder();
+        $qb  = $this->getEntityManager()->getConnection()->createQueryBuilder();
 
         return $qb->select('*')
-            ->from(MAUTIC_TABLE_PREFIX.'lead_devices', 'es')
-            ->where('lead_id = :leadId')
-            ->setParameter('leadId', (int) $lead->getId())
-            ->orderBy('date_added', 'desc')
-            ->execute()
-            ->fetchAll();
+        ->from(MAUTIC_TABLE_PREFIX.'lead_devices', 'es')
+        ->where('lead_id = :leadId')
+        ->setParameter('leadId', (int) $lead->getId())
+        ->orderBy('date_added', 'desc')
+        ->execute()->fetchAllAssociative();
     }
```

Comment:
Most of the changes here are unnecessary and for the worse. Was there some reason for this style change?

Reply:
@escopecz I am not sure why `CSFixer` did not take care of that, I solved it anyways and all the other styling issues

Reply:
Thanks!